script.js

```diff
@@ -52,33 +51,73 @@ let hondaCars = [
   },
 ];
 
-function search(make) {
-  let carMake = document.getElementById("car-make").value;
-  let carPlate = document.getElementById("car-plate").value;
+let list = document.getElementById("list");
 
-  if (carMake == "Ford" && carPlate == 27360) {
-    document.getElementById("carPlateDisplay").innerText = fordCars[0].plate;
-    document.getElementById("carModelDisplay").innerText = fordCars[0].model;
-    document.getElementById("carYearDisplay").innerText = fordCars[0].year;
-    document.getElementById("carYearDisplay").innerText = fordCars[0].year;
-    document.getElementById("carPoliza").innerText = fordCars[0].poliza.number;
+function setList(group) {
+  clearList();
+  for (const car of group) {
+    const item = document.createElement("li");
+    const text = document.createTextNode(
+      "Brand: " +
+        car.brand +
+        " - " +
+        " Plate: " +
+        car.plate +
+        " Model: " +
+        car.model
```

Comment:
instead of concatenating strings with variables, use [template literals](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Template_literals)